src/gridDynMain/CMakeLists.txt

```diff
@@ -19,7 +19,7 @@ set(GRIDYNMAIN_BINARY_LOC ${CMAKE_CURRENT_BINARY_DIR} PARENT_SCOPE)
 message(STATUS "execOnly:${executable_only_components}") # TODO what even are these?
 
 # TODO why wasn't fmiBuilder required before?
-target_link_libraries(gridDynMain ${executable_only_components} runner fmiBuilder)
```

Comment:
fmiBuilder should not be a requirement,  it should be part of the exectuable only_components if the FMI_EXPORT option is enabled.

Reply:
Okay, that makes more sense. executable_only_components has been broken since at least June, I'll try and get that fixed too. I'm still not sure why this was working earlier, though.